testcases/open_posix_testsuite/conformance/interfaces/mmap/assertions.xml

```diff
@@ -134,12 +134,6 @@ or fildes is not open for write and PROT_WRITE was specified for a
 MAP_SHARED type mapping.
 	  </assertion>
 
-      <assertion id="18" tag="ref:XSH6:25310:25315">
-The mmap( ) function shall fail if:
-ML [EAGAIN] The mapping could not be locked in memory, if required by mlockall ( ), due to
-a lack of resources.
-      </assertion>
-
       <assertion id="19" tag="ref:XSH6:25310:25342">
 The mmap( ) function shall fail if:
 [EBADF] The fildes argument is not a valid open file descriptor.
```

Comment:
There is nothing wrong about this assertion (it does not even mention the rlimit), hence there is no need to remove it.